department-of-veterans-affairs/va.gov-cms
# VACMS-1421: Replace non rich text intro field with rich text intro.

## Description
See #1421 
**DO NOT MERGE PRIOR TO https://github.com/department-of-veterans-affairs/vets-website/pull/15880 (merged) being released to prod.**

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/106646214-76883780-655b-11eb-8081-784cfd866af9.png)

## QA steps
As an administrator, go to `/node/993/edit` and visually verify that the page introduction field is wysiwyg

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
